app/amity.py

```diff
@@ -1,33 +1,413 @@
+        if filename:
+
+            with open(filename, 'r') as people_file:
+                for line in people_file:
+                    information = line.split()
+                    first_name = information[0]
+                    second_name = information[1]
+                    full_name = first_name + " " + second_name
+                    description = information[2]
+                    if len(information) == 4:
+                        wants_accommodation = information[3]
+                    else:
+                        wants_accommodation = "N"
+                    self.add_person(person_name=full_name,
+                                       person_description=description,
+                                       wants_accommodation=wants_accommodation)
+                    print("adding people...")
+
+        else:
+            print("please provide a text file")
+
+
+
+
+
+
+    def employee_position(self, name):
+        """defines position of person"""
+        for person in self.fellows_list:
+            if person.person_name == name:
+                return "IsFellow"
+        for person in self.staff_list:
+            if person.person_name == name:
+                return "IsStaff"
+
+
+    def reallocate_person(self, person_name, new_room, rtype):
+        """reallocates person allocates person to a specified room """
+
+        available = [room.room_name.upper() for room in self.all_rooms if room.occupants < room.max_occupants ]
+        person_found = False
+        for person in self.all_people:
+            if person.person_name == person_name:
+                person_found = True
+                break
+        if not person_found:
+            print("Person does not Exist")
+            return
+
+        if person_found:
+            if rtype == "lspace":
+
+                current_room = [room_name ]
+                if self.employee_position(person_name) == "IsStaff":
+                    print("sorry, staff cannot be assigned living spaces")
+                elif self.employee_position(person_name.upper()) == "IsFellow":
+                    if new_room.upper() not in available:
+                        print("sorry, the room is not available")
+                    else:
+                        reallocated_lspace = new_room
+                        self.lspace_allocations[new_room].append(person_name.upper())
+                        print("congratulations %s, you have been re-assigned to room %s"
+                              %(person_name, reallocated_lspace))
+
+            else:
+                if new_room.upper() not in available:
+                    print("sorry, the room is not available")
+                else:
+                    reallocated_office = new_room
+                    print("congratulations %s, you have been re-assigned to room %s"
+                          %(person_name, reallocated_office))
+
+
+
+
+    def print_office_allocations(self, filename=None):
```

Comment:
The two print functions are doing the same thing. Just refactor them into one function.